### DISCUSSION TITLE
Google Login in Ionic Angular App

### DISCUSSION THREAD

**Discussion by michel-silva** [capawesome-team/capacitor-firebase#584] [Q&A]
Hello,

I'm trying to use this package to build an app with Google login with Firebase.
For the web, all works fine, but for the Android app, it's not working.

I created a new topic on the ionic framework forum, there I added more details about my problem.
[https://forum.ionicframework.com/t/ionic-cordova-google-login/240250](https://forum.ionicframework.com/t/ionic-cordova-google-login/240250)

Could someone help me with this problem?

Any suggestions are welcome.

Thank you.

**Comment by robingenz** [comment]
This is not enough information to help you. Something is wrong with your configuration. Feel free to have a look at our [official demo app](https://github.com/robingenz/capacitor-firebase-authentication-demo). You might also find this [Android setup guide](https://docs.ionstarter.dev/angular-firebase/setup/android/) from [Ionstarter](https://ionstarter.dev/) helpful.
